internal/provider/blueprint_resource.go

```diff
@@ -155,6 +156,14 @@ func (r *BlueprintResource) Schema(
 					),
 				},
 			},
+			"is_published": schema.BoolAttribute{
+				Computed:            true,
+				Optional:            true,
+				MarkdownDescription: "A published blueprint is available for use by developers to create resources through the Resourcely portal.\n\nIf left unset, you may safely change this property in the Resourcely portal. In this case, the blueprint will start as unpublished.",
```

Comment:
total bikeshedding...

```suggestion
				MarkdownDescription: "A published blueprint is available for use by developers to create resources through the Resourcely portal.\n\nIf left unset, the blueprint will start as unpublished, and you may safely change this property in the Resourcely portal.",
```

Reply:
Beautiful, thanks!